src/coreclr/inc/nsutilpriv.h

```diff
@@ -159,44 +159,6 @@ void MakePath(                          // throws on out of memory
     const SString &ssNameSpace,         // Namespace for name.
     const SString &ssName);             // Final part of name.
 
-//*****************************************************************************
-// Concatinate type names to assembly names
-//*****************************************************************************
-static
-bool MakeAssemblyQualifiedName(                                  // true if ok, false if out of memory
```

Comment:
Also delete the implementation in https://github.com/dotnet/runtime/blob/main/src/coreclr/utilcode/namespaceutil.cpp